daemon/protocol/service.go

```diff
@@ -81,3 +83,60 @@ func DriverStates() *DriverStatesResponse {
 
 	return resp
 }
+
+//proteus:generate
+type InstallDriverRequest struct {
+	// Language supported by the driver being installed.
+	Language string
+	// ImageReference name of the image to be installed in the following format:
+	// `[transport://]name[:tag]` if transport is missing `docker` or tag is
+	// missing `lastest` are assumed.
+	ImageReference string
+	// Update if update if false install fails if an previous image already
```

Comment:
`// Update indicates whether an image should be updated. When set to false, the installation fails if the image already exists.`